web-app/django/VIM/apps/instruments/management/commands/download_imgs.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import os
+import requests
+from PIL import Image
+from io import BytesIO
+from django.core.management.base import BaseCommand

```

Comment:
Are you using pylint (in the dev dependencies)? It would alert you that your imports are in non-standard order.

Reply:
Yes I'm using it. I'll check and modify the code followed by pylint's suggestions.